wp-content/civi-extensions/goonjcustom/Civi/Traits/QrCodeable.php

```
@@ -23,21 +24,192 @@ public static function generateQrCode($data, $entityId, $saveOptions) {
+      // --- Step 4: Venue text with wrapping
+      $bottomFontSize = 25;
+      $maxWidth = $canvasWidth - 80;
+      $bottomY = $qrY + $newQrHeight + 40;
+
+      // Start with bold "Venue:".
+      $bboxLabel = imagettfbbox($bottomFontSize, 0, $fontPath, $venueLabel);
+      $labelWidth = abs($bboxLabel[2] - $bboxLabel[0]);
+
+      $lines = [];
+      $currentLine = $venueLabel;
+      $currentWidth = $labelWidth;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$currentWidth'. (Unused Code Rules)

(UnusedLocalVariable)